csv_provider/README.md

```diff
@@ -7,7 +7,7 @@ The provider requires an installation of Python in version 3 and can be executed
 
 ```
 pip install -r requirements.txt
-python provider.py
+python3 provider.py
```

Comment:
@SebastianSchildt - I have noticed that we are quite inconsistent regarding if we use `python` or `python3` in our examples. We definitively cannot use "python2" but in most distros "python2" isn't available anyway, and it might be so that your `python` command anyway points to a `python3` installation. We also likely do not support older 3.X-versions anyway.

But in short - do we prefer to use `python` or `python3` in examples? If we agree on preferred term I have an idea of aligning the examples so that they all use the same term. A similar discussion exists for pip vs pip3.

Reply:
Personally I prefer python3 because I use it my own but no „strong“ opinion

Reply:
Just saw the MS Teams bot ate my answer, I wrote this

No strong opinion either. Asking me like this I'd prefer just "python", I do have a vague memory that some of the python3 where needed in the past, as the default was Py2 on some distros also used in containers. But that may have been ancient Ubuntu 16.04 times, so I'd be fine to use "pip" and "python", if container builds break we will see it. From user perspective, depending what environment they use, they can fall into a hole either way....